src/test/java/uk/gov/hmcts/reform/orgrolemapping/controller/testingsupport/PrmSchedulerControllerTest.java

```diff
@@ -93,4 +93,31 @@ void process3Test_noParam() {
         process3Test(null);
     }
 
+    @Test
+    void process5Test() {
+        process5Test("2023-10-01T13:40:03");
+    }
+
+    private void process5Test(String since) {
+        ProcessMonitorDto processMonitorDto = new ProcessMonitorDto("Test Process5");
+
+        when(organisationService.getBatchLastRunTimestampEntity()).thenReturn(batchLastRunTimestampEntity);
+
+        if (since != null) {
+            batchLastRunTimestampRepository.save(batchLastRunTimestampEntity);
```

Comment:
This call is to a mock so I assume will have no affect on outcome. Can we use a verify to check if save is called (or not called) when value is set rather than this?  (repeat change for previous similar `process3Test(String since)`).

Reply:
Resolved in 46dadb1822fbb407102b2a38536dc533d6d259da.